package.json

```diff
@@ -29,6 +31,7 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
+    "update": "npx msw init public --no-save",
```

Comment:
Why the scripts is set as `update` here? Maybe a more specific name will be better.